deploy/script/katello-restore

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# Echo all commands
+usage(){
+  echo "Usage: katello-restore <backup directory>"
+  echo " eg: $ katello-restore /var/tmp/katello-backup-201501261543"
+}
+
+if [ "$#" -lt 1 ]; then
+    echo "ERROR: Please specify an export directory"
+    usage
+    exit 1
+fi
+
+if [[ "$2" != "-y" ]]; then
+  echo ""
+  echo "WARNING: This script will drop and restore your database."
+  echo "Your existing installation will be replaced with the backup database."
+  echo ""
+  echo "Once this operation is complete there is no going back."
+  echo ""
+  read -p "Are you sure(Y/N)? " -n 1 -r
+  echo ""
+  if [[ ! $REPLY =~ ^[Yy]$ ]]; then
+      echo "** cancelled **"
+      exit 1
+  fi
+fi
+
+export BDIR=$1
+chgrp postgres -R $BDIR
+cd $BDIR
+
+# Reset katello
+echo "Resetting Katello..."
+katello-installer --reset
+echo "Done."
+echo ""
+
+# Restore system files
+echo "Restoring system files..."
+tar --selinux -xzvf config_files.tar.gz -C /
```

Comment:
This needs to be done before the installer is run or else you may over write anything that the installer configured and then there will be a mis-match. By laying this down first, the installer will correct any configuration that might be out of sync with expectations and provide a more robust restoration.